core/conn/jdbc_type2/native/SQLMXConnection.cpp

```diff
@@ -590,6 +590,20 @@ JNIEXPORT void JNICALL Java_org_trafodion_jdbc_t2_SQLMXConnection_connectInit
         strcpy(srvrGlobal->CurrentSchema, "SEABASE");
 
     }
+ 
+    odbc_SQLSvc_SetConnectionOption_sme_(NULL, NULL,
+                 &setConnectException,
+                 dialogueId,
+                 BEGIN_SESSION,
+                 0,
+                 NULL,
```

Comment:
Consider passing in client_name and  thread_id  so that it is identifiable on the server side. This optional session name has a limit of 24 characters